cmd/manager/main.go

```diff
@@ -79,56 +87,61 @@ func start() error {
 	}
 
 	ctx := context.TODO()
-
-	// Become the leader pod if within namespace. If outside a cluster, skip
-	// and return nil.
+	// Become the leader before proceeding
 	err = leader.Become(ctx, "certman-operator-lock")
 	if err != nil {
 		log.Error(err, "")
-		return err
+		os.Exit(1)
+	}
+
+	// Set default manager options
+	options := manager.Options{
+		Namespace:          namespace,
+		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
```

Comment:
Setting up the sdk default Metrics here conflicts with our custom metrics server that's defined below. Removing line 109 allows the operator to run as expected in test environments.